### DISCUSSION TITLE
Q-Select intercept add and remove actions

### DISCUSSION THREAD

**Discussion by gregory-h** [quasarframework/quasar#12965] [General - Components / Directives / etc]
I am using a q-select with chips, an [<object>] v-model, map-options, autocomplete and lazy filtering.

This works fabulously, but when an item is added via the filter menu I would like to be able to intercept the add, make an api call, then complete or cancel the add. Similarly when an entry is removed using the chip (x) button I would like to be able to intercept and then asynchronously complete or block the remove. 
 
Looking thru the api's I don't see a way to do this; I see the add and remove events but would like to intercept before the operation occurs and conditionally complete or cancel. Wondering if there is a way to achieve this as is? 

Awesome component btw.

**Comment by MilosPaunovic** [comment]
Check this out: [https://github.com/quasarframework/quasar/discussions/12659](https://github.com/quasarframework/quasar/discussions/12659)
